circuit_tutorials/jolt/sha3_chain/src/utils.rs

```diff
@@ -0,0 +1,80 @@
+use ark_serialize::{CanonicalDeserialize, CanonicalSerialize};
+use base64::{engine::general_purpose, Engine as _};
+use serde::{Deserialize, Serialize};
+
+use common::rv_trace::ELFInstruction;
+use jolt::{
+    field::JoltField,
+    jolt::vm::{rv32i_vm::RV32IJoltProof, JoltCommitments},
+    poly::commitment::commitment_scheme::CommitmentScheme,
+};
+
+// This struct is used to store the base64 encoded proof data generated by
+// Sindri.
+#[derive(Debug, Serialize, Deserialize)]
+pub struct JsonProofData {
+    pub jolt_proof: String,
+    pub jolt_commitments: String,
+    pub bytecode: String,
+    pub memory_init: String,
+}
+
+// This struct stores the JoltProof and JoltCommitments structs.
+#[derive(CanonicalSerialize, CanonicalDeserialize)]
+pub struct JoltProofStruct<F, PCS>
+where
+    F: JoltField,
+    PCS: CommitmentScheme<Field = F>,
+{
+    pub proof: RV32IJoltProof<F, PCS>,
+    pub commitments: JoltCommitments<PCS>,
+}
+
+// This struct stores the bytecode and memory_init data used to recreate the
+// preprocessing struct.
+#[derive(Serialize, Deserialize)]
+pub struct PreprocessingStruct {
+    pub bytecode: Vec<ELFInstruction>,
+    pub memory_init: Vec<(u64, u8)>,
+}
+
+// This function is deserializes the base64 encoded proof data and returns the
```

Comment:
remove "is"

Reply:
done [ca75ce8](https://github.com/Sindri-Labs/sindri-resources/pull/100/commits/ca75ce80870c5518d57e3bb6070b2a2a5ee54565)